### DISCUSSION TITLE
Combined minimumReleaseAge with major updates only

### DISCUSSION THREAD

**Discussion by greg64230** [renovatebot/renovate#27186] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Is it posible to apply the configuration option minimumReleaseAge for major updates only ?

I'm using renovate to update npm and maven dependencies and I want to apply this configuration for both.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Yes with matchUpdateTypes=major

**Reply by greg64230** [reply]
Thank you for your answer.

Something like this ?

`{
  "packageRules": [
    {
      "matchUpdateTypes": ["major"],
      "minimumReleaseAge": "30 days"
    }
  ]
}`
